demo/agilebank/bad_resources/opa_wrong_repo_namespaceselector.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
This looks to be the same resource as `opa_wrong_repo.yaml` is there some way we can make it specific to the namespace selector constraint?

Also, because the demo script relies on applying all constraints/templates in their respective directories, maybe we need another playground directory where people can just add/remove constraints and templates they find interesting that wont affect the demo.sh scripts.

Reply:
moved to a new `example` folder